src/main/kotlin/org/elm/ide/intentions/exposure/ExposeIntention.kt

```diff
@@ -19,37 +20,40 @@ open class ExposeIntention : ExposureIntentionBase<ExposeIntention.Context>() {
     override fun findApplicableContext(project: Project, editor: Editor, element: PsiElement): Context? {
         val exposingList = getExposingList(element) ?: return null
 
-        // check if the caret is on the identifier that names the exposable declaration
         val decl = element.parent as? ElmExposableTag ?: return null
         if (decl.nameIdentifier != element) return null
 
         return when {
-            decl is ElmUnionVariant -> {
-                // might be nice to support this in the future (making a union type fully exposed)
-                null
-            }
-
-            decl is ElmFunctionDeclarationLeft && !decl.isTopLevel ->
-                null
-
-            !exposingList.exposes(decl) ->
-                createContext(decl, exposingList)
-
-            else ->
-                null
+            // might be nice to support this in the future (making a union type fully exposed)
+            decl is ElmUnionVariant -> null
+            decl is ElmFunctionDeclarationLeft && !decl.isTopLevel -> null
+            !exposingList.exposes(decl) -> createContext(decl, exposingList)
+            else -> null
         }
     }
 
-    /**
-     * Creates a [Context] based on the passed in parameters. Overriding subclasses can return null if they find that
-     * the passed in [decl] isn't valid for their particular intention.
-     */
```

Comment:
Was there nothing useful in this comment. I guess it explains why the return type has a `?`?